superset-frontend/src/explore/components/ControlPanelsContainer.tsx

```diff
@@ -431,6 +466,32 @@ export const ControlPanelsContainer = (props: ControlPanelsContainerProps) => {
     [handleClearFormClick, handleContinueClick, hasControlsTransferred],
   );
 
+  const dataTabTitle = useMemo(
+    () => (
+      <>
+        <span>{t('Data')}</span>
+        {props.errorMessage && (
+          <span
+            css={(theme: SupersetTheme) => css`
+              font-size: ${theme.typography.sizes.xs}px;
+              margin-left: ${theme.gridUnit * 2}px;
+            `}
+          >
+            {' '}
+            <Tooltip
+              id="query-error-tooltip"
+              placement="right"
+              title={props.errorMessage}
+            >
+              <i className="fa fa-exclamation-circle text-danger fa-lg" />
```

Comment:
Shouldn't we use an icon from the `Icons` component?

Reply:
That's the same icon as the one that's displayed next to controls. I agree that we should drop font-awesome icons in favor of antd, but changing that here would also require changing the danger icon in other places in control panel. Let's do that in a separate PR!